crates/hyperswitch_domain_models/src/router_request_types.rs

```diff
@@ -629,6 +621,19 @@ pub struct RefundIntegrityObject {
     pub refund_amount: MinorUnit,
 }
 
+#[derive(Debug, serde::Deserialize, Clone)]
+pub enum SplitRefundsRequest {
+    StripeSplitRefund(StripeSplitRefund),
+}
+
+#[derive(Debug, serde::Deserialize, Clone)]
+pub struct StripeSplitRefund {
+    pub charge_id: String,
+    pub transfer_account_id: String,
```

Comment:
Does the account ID contain sensitive information? Should we consider wrapping this within a `Secret`?

Reply:
The account ID is used to differentiate between the account to which the money needs to be transferred, whether it is the platform account or the connected account.

Reply:
That's okay, but we don't need it to be included (in plaintext) in our logs right?

Reply:
Yes, we can log it as plain text since it is similar to business_profile and does not contain inherently sensitive information.